androidApp/MovieRecommendationApp/app/src/main/java/com/example/movierecommendationapp/LoginFragment.java

```diff
@@ -33,13 +34,21 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         emailEditText = view.findViewById(R.id.emailEditText);
         passwordEditText = view.findViewById(R.id.passwordEditText);
 
+        loginBtn.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Intent intent = new Intent(getContext(),ProfileActivity.class);
+                startActivity(intent);
+            }
+        });
+
         clearBtn.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
                 AlertDialog.Builder alertDialog = new AlertDialog.Builder(getActivity());
                 alertDialog.setTitle("Clear Fields");
                 alertDialog.setMessage("Are you sure to clear?");
-                alertDialog.setIcon(R.drawable.ic_baseline_warning_24);
+                alertDialog.setIcon(R.drawable.ic_launcher_foreground);
```

Comment:
Revert the icon back to `ic_baseline_warning_24`

Reply:
```suggestion
                alertDialog.setIcon(R.drawable.ic_launcher_foreground);
```
Committed the above